checks/http/google.rb

```diff
@@ -125,6 +125,19 @@ def generate_checks(url)
           :paths => ["#{url}"],
           :inference => false
         },
+        {
+          :type => "fingerprint",
+          :category => "application",
+          :tags => ["Web Server", "Hosting"],
```

Comment:
This should be tagged SaaS - There are four tags that are now used to determine if an entity is "cloud hosted" - https://github.com/intrigueio/intrigue-core/blob/3b3019fcec031c9dfbb0baa660ae59963e1f0b42/lib/tasks/helpers/cloud_providers.rb#L121 (SaaS, IaaS, PaaS, and  "Cloud") ... these are important for being able to flag cloud assets.